### ISSUE TITLE
Error when using createLog in IE11

### ISSUE THREAD

**Issue by srijken** [exceptionless/Exceptionless.JavaScript#69]
This was introduced with #22, take care that whatever that PR fixed doesn't get reintroduced by fixing this.

```
TypeError: Accessing the 'caller' property of a function or arguments object is not allowed in strict mode
at ExceptionlessClient.prototype.createLog() in https://cdn.rawgit.com/exceptionless/Exceptionless.JavaScript/v1.0.1/dist/exceptionless.js:line 2082:col 13
at S() in https://****.****.net/scripts/app.min.js:line 23:col 2530
at C() in https://****.****.net/scripts/app.min.js:line 23:col 1023
at c() in https://****.****.net/scripts/app.min.js:line 29:col 20751
at Anonymous function() in https://****.****.net/scripts/app.min.js:line 29:col 29486
at Anonymous function() in https://cdnjs.cloudflare.com/ajax/libs/angular.js/1.4.6/angular.min.js:line 118:col 324
at n.prototype.$eval() in https://cdnjs.cloudflare.com/ajax/libs/angular.js/1.4.6/angular.min.js:line 132:col 441
at n.prototype.$digest() in https://cdnjs.cloudflare.com/ajax/libs/angular.js/1.4.6/angular.min.js:line 129:col 455
at n.prototype.$apply() in https://cdnjs.cloudflare.com/ajax/libs/angular.js/1.4.6/angular.min.js:line 133:col 234
at g() in https://cdnjs.cloudflare.com/ajax/libs/angular.js/1.4.6/angular.min.js:line 87:col 374
at K() in https://cdnjs.cloudflare.com/ajax/libs/angular.js/1.4.6/angular.min.js:line 91:col 448
at z.onload() in https://cdnjs.cloudflare.com/ajax/libs/angular.js/1.4.6/angular.min.js:line 92:col 462
```

**Comment by niemyjski** [comment]
we need to detect strict mode and just not even call this code: http://stackoverflow.com/questions/10480108/is-there-any-way-to-check-if-strict-mode-is-enforced

**Comment by srijken** [comment]
This was fixed in 1.4.3 with https://github.com/exceptionless/Exceptionless.JavaScript/commit/210ef6e38440525e72ca7aed3e7961842144020e
